src/app/core/services/split-expense.service.ts

```
@@ -415,4 +417,104 @@
 
     return this.expenseCommentService.post(commentsPayload);
   }
+
+  normalizeSplitAmounts(splitExpensesFormArray: UntypedFormArray, targetAmount: number, currency: string): void {
+    if (splitExpensesFormArray.length === 0) {
+      return;
+    }
+
+    const precision = this.getCurrencyPrecision(currency);
+    const threshold = this.getThresholdTolerance(precision);
+    const roundedTargetAmount = parseFloat(targetAmount.toFixed(precision));
```

Biome (1.9.4)

[error] Use Number.parseFloat instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.parseFloat instead.


(lint/style/useNumberNamespace)